app/scripts/controllers/journalUpload.js

```diff
@@ -11,7 +11,7 @@ angular
     $window,
     $cookies,
     $uibModal,
-    registrations,
+    registrationsData,
```

Comment:
I figured ```registrationsData``` would be a better name instead of having to do ```registrations.registrations```. Let me know what you think.